ckan/tests/logic/action/test_create.py

```diff
@@ -2001,5 +2001,9 @@ def test_ignored_on_create_if_non_sysadmin(self):
             'package_create', context=context, **pkg_dict
         )
         assert "plugin_data" not in created_pkg
-        pkg = model.Package.get(created_pkg["id"])
-        assert pkg.plugin_data is None
+
+        plugin_daya_from_db = model.Session.execute(
```

Comment:
daya?

Reply:
It's a typo, tnx, good catch btw the failed test is not related to my changes